src/ConfigWidgets/SymbolsDialog.cpp

```diff
@@ -31,22 +32,86 @@ constexpr const char* kOverrideWarningText =
 
 using orbit_client_data::ModuleData;
 using orbit_grpc_protos::ModuleInfo;
+using orbit_object_utils::SymbolsFile;
+
+namespace {
+class MappingItem : public QListWidgetItem {
```

Comment:
A comment for the class would be nice. It also might be nice to have the word Override or Force in the class name to make it more clear that this is only used when the user specifically overrides something? But I understand if you don't want to make the name too long, just a thought.

Reply:
I called it `OverrideMappingItem` and added a class description comment. I am not really concerned about naming length, so maybe a even more descriptive name makes sense... maybe `ModuleSymbolFileOverrideItem`.. wdyt?

Reply:
I don't have a strong preference but OverrideMappingItem sounds good to me.